pkg/agent/clusterclaim.go

```diff
@@ -0,0 +1,97 @@
+package agent
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/stolostron/hypershift-addon-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	clusterclientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+
+	clusterv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+)
+
+const (
+	// labelExcludeBackup is true for the local-cluster will be not backed up into velero
+	labelExcludeBackup = "velero.io/exclude-from-backup"
+
+	hypershiftManagementClusterClaimKey = "managementcluster.hypershift.openshift.io"
+	hypershiftHostedClusterClaimKey     = "hostedcluster.hypershift.openshift.io"
+)
+
+func newClusterClaim(name, value string) *clusterv1alpha1.ClusterClaim {
+	return &clusterv1alpha1.ClusterClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
+			Labels: map[string]string{labelExcludeBackup: "true"},
```

Comment:
why this should be labled to backup?

Reply:
I want it *exclude* from backup, @zhiweiyin318 do we need this here?

Reply:
yes. I think the label is to exclude resources from backup